ipatests/test_integration/test_installation.py

```diff
@@ -703,6 +703,30 @@ def install(cls, mh):
     def test_install_master(self):
         tasks.install_master(self.master, setup_dns=False)
 
+    def test_smoke_test_for_debug_mode(self):
+        """Test if an IPA server works in debug mode.
+        Related: https://pagure.io/freeipa/issue/8891
+        """
+
+        target_fname = paths.IPA_SERVER_CONF
+        assert not self.master.transport.file_exists(target_fname)
+
+        # set the IPA server in debug mode
+        server_conf = "[global]\ndebug=True"
+        self.master.put_file_contents(target_fname, server_conf)
+        assert self.master.transport.file_exists(target_fname)
```

Comment:
Is it really needed? We put the file in previous action and assume `put_file_contents` does its job.

Reply:
This is not 100% needed. I can remove it if you like.

Reply:
Removed, thanks.